mandatory/2-function-creation.js

```diff
@@ -4,7 +4,13 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
```

Comment:
Nice 👍 
Can you think about how you can make this code even shorter? Do you need the `if` statement? Can you do it with only a `return` statement?